libs/damap/src/lib/components/dmp/legal-ethical-aspects/legal-ethical-aspects.component.html

```diff
@@ -65,14 +65,12 @@ <h2 translate>dmp.steps.legal.heading</h2>
           <app-input-wrapper
             [label]="'dmp.steps.legal.question.otherDataSecurityMeasures'"
             [control]="otherDataSecurityMeasures"
-            [maxLength]="4000"
-            appearance="fill"></app-input-wrapper>
+            [maxLength]="4000"></app-input-wrapper>
```

Comment:
When adding **maxLength** to the **app-input-wrapper**, it’s better to specify which forms should have it, rather than using different forms for each case. We have now for _app-input-wrapper_ with _maxLength_ 4000 two different forms how we represent them. Would go with one of them. 

Example:
![Distinguish between textwrapper and new-form1](https://github.com/user-attachments/assets/5e8a7563-d137-479d-a465-54db792bceee)
![Distinguish between textwrapper and new-form](https://github.com/user-attachments/assets/3b31e399-4f19-4e0e-b673-3e955ee5e47d)

Reply:
What do you mean with two different forms?
The picture you sent uses app-textarea-wrapper, do you want to say that we could use app-input-wrappers with maxLength=4000 insted. Please correct em if i am wrong.

Reply:
For the input wrapper maxLength, we previously showed the ‘filled’ view, and in the Legal Aspects we only changed this for the Legal Aspects.

Previously, the changes looked like the rest:
<img width="1129" alt="Screenshot 2024-10-21 at 15 23 29" src="https://github.com/user-attachments/assets/f29ad7c0-04e5-4721-af34-4d4fff22dc77">

Reply:
I will change all occurences of app-input-wrapper with maxLength 4000 to app-textarea-wrapper, to keep it consistent in the whole application. app-input wrapper is definitely the wrong component here, with 4000 chars you need more than one line for sure.